umbraco-cloud/product-upgrades/major-upgrades.md

```diff
@@ -76,20 +76,12 @@ Look for the "**Upgrade from/to Umbraco xx"** boxes. These boxes contain importa
 
 ## Step 3: Upgrade the project locally using Visual Studio
 
-1. Open your project in Visual Studio - use the `csproj` file in the `/src/UmbracoProject` folder.
-2. Right-click your project solution in **Solution Explorer**.
-3. Select **Properties**.
-
-<figure><img src="images/Solution-Explorer.png" alt=""><figcaption></figcaption></figure>
-
-4. Change the **Target framework** in the **General** section of the **Application** tab.
-   1. Choose the version you set on your Cloud environment in [Step 1](major-upgrades.md#step-1-enable-.net).
-
-![Target Framework](images/Target-Framework.png)
-
-5. Go to **Tools** > **NuGet Package Manager** > **Manage NuGet Packages for Solution.**
-6. Navigate to the **Updates** tab.
-7. Select the version you are updated to, and follow the instructions:
+1. Open the `csproj` file located in the `/src/UmbracoProject` folder.
+2. Right-click your project in **Solution Explorer** and select **Properties**.
+3. Change the **Target framework** under the **Application** tab to match the version set in your Cloud environment from [Step 1](major-upgrades.md#step-1-enable-.net).
```

Comment:
I'm wondering if it makes sense, to change these steps a bit.
So, instead of opening the properties, we open the .csproj file in Visual Studio.
There you can also change the .NET version.

If we do this, we can move the part about the package references that needs to be deleted in 15 - we can move these into an expandable under these bullets, as it's changes that needs to be made in the same file 🤔 

Currently, the steps for deleting, needs to be done after a step that says you need to open the NuGet package manager - which I guess is not really a good workflow 🤔